src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/DataHelper.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.hmcts.cmc.claimstore;
+
+import uk.gov.hmcts.cmc.claimstore.processors.JsonMapper;
+import uk.gov.hmcts.cmc.claimstore.repositories.mapping.JsonMapperFactory;
+import uk.gov.hmcts.cmc.domain.utils.ResourceReader;
+import uk.gov.hmcts.document.domain.UploadResponse;
+
+public class DataHelper {
+    private static final JsonMapper jsonMapper = JsonMapperFactory.create();
```

Comment:
can we use https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/test/autoconfigure/json/JsonTest.html
instead?